pysatSeasons/avg.py

```diff
@@ -10,12 +10,14 @@
 
 import numpy as np
 import pandas as pds
+import warnings
 
 import pysat
 import pysatSeasons as pyseas
 
 
-def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=False):
+def median1D(const, bin1, label1, data_label, auto_bin=True, returnData=None,
```

Comment:
Should the upstream docstrings also note the deprecation by having the deprecation warning snippet up top?